applications/portal/backend/api/models/experiment.py

```diff
@@ -78,6 +81,36 @@ def save(self, force_insert=False, force_update=False, using=None, update_fields
         super(Experiment, self).save(force_insert, force_update, using, update_fields)
         create_dir_if_not_exists(self.storage_path)
 
+    def register(self, filepath, populations):
+        if not populations:
+            return
+
+        is_fiducial = populations[0].is_fiducial
+
+        Population = apps.get_model('api', 'Population')
```

Comment:
question: @afonsobspinto isn't it better to move this into the/a "service" layer so that we can use the import instead of `apps.get_model`